decidim-core/app/cells/decidim/diff/diff_split.erb

```diff
@@ -0,0 +1,11 @@
+<div class="mediumlarge-6 columns">
+  <div class="diff-direction-label">
+    <%= t("versions.directions.#{direction}") %>
```

Comment:
Not super important, but I prefer using this:

```suggestion
    <%= t(direction, scope: "versions.directions") %>
```

Reply:
Thanks.